keyboards/handwired/dactyl_manuform/5x6_5/keymaps/cykedev/config.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Christian Eiden, cykedev
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#undef SOFT_SERIAL_PIN
+#undef DIODE_DIRECTION
+#undef MATRIX_ROW_PINS
+#undef MATRIX_COL_PINS
+#undef BOOTMAGIC_LITE_ROW
+#undef BOOTMAGIC_LITE_COLUMN
+#undef BOOTMAGIC_LITE_ROW_RIGHT
+#undef BOOTMAGIC_LITE_COLUMN_RIGHT
+
+
+#define USE_SERIAL
+#define SOFT_SERIAL_PIN D2
+#define EE_HANDS
+#define SPLIT_USB_DETECT
```

Comment:
Are you using a v3 Elite C? Or a Teensy?

Reply:
no, just a standard ProMicro. Without SPLIT_USB_DETECT I did not get the two halfes to work, only one.

Reply:
That's odd.  The Split USB Detect should only be for controllers that don't have proper VBUS detection.  The Pro Micros do (or at least, should).   The Elite C v3 had a design flaw that caused it to not work, and the teensy's don't support it without a hack. 

That's why I asked.  In theory, the pro micro shouldn't need it.  

Though, are you hooking up power via the RAW pin on the pro micros?

Reply:
> I think I did, connected to GND, VCC and D2. I had a note from the shop I bought the ProMicro from on the bag, that told me that this batch had issues and SPLIT_USB_DETECT is needed to get them to work -.-

They probably had bad/incorrectly rated diodes, which would lead to issues, yeah. 

No problem, then.